packages/components/organisms/f-footer/test/visual/f-footer.visual.spec.js

```diff
@@ -0,0 +1,69 @@
+import Footer from '../../test-utils/component-objects/f-footer.component';
```

Comment:
Old desktop / mobile tests were deleted and both moved into the same file to match other components.